profile/README.md

```diff
@@ -1,2 +1,118 @@
+**Location Data**: Limited to users in the Netherlands.
+
+## Project Timeline
+
+**Start Date**: 16 August 2023<br>
+**End Date**: December 2023
+
+Development Phases:
+1. Ideation Phase<br>
+2. Initiative Phase<br>
+3. Definition Phase<br>
+4. Design Phase<br>
+5. Preparation Phase<br>
+6. Realization Phase<br>
+7. Maintenance Phase (Post-launch)
+
+## Goals
+
+The primary objective of this project is to develop an application that is:
+
+**Cross-Platform**: Compatible with both Apple and Samsung devices.<br>
+**User-Friendly**: Ergonomically designed to ensure ease of use.<br>
+**Cost-Efficient**: Helps users save money on fuel through intelligent route planning and price comparison.
+
+## Project Boundaries
+
+**App Compatibility**: The app will be compatible with iOS 17.0.3+ and Android 12+ devices.<br>
+**Geographical Scope**: Location-based services will be limited to the Netherlands.<br>
+**Core Features**: Route planning, fuel price comparison, station details, and user guide.
+
+## Deliverables
+
+Upon completion, the following deliverables will be provided to Compairifuel:
+
+A fully functional cross-platform app for both Apple and Samsung devices.<br>
+An ergonomic and user-friendly design.<br>
+A comprehensive user guide integrated into the app for ease of use.<br>
+An app that effectively helps users reduce fuel costs through better route planning and station selection.
+
+## Development Methodology
+
+The app development will follow an Agile methodology, with sprints being applied during the design and realization
+phases. Regular sprint reviews and sprint planning sessions will be held to monitor progress and adjust development
+priorities.
+
+## Project Team
+
+**Project Leader**: Jaron Zijlstra<br>
+**Client**: Jelle Schaafsma<br>
+**Testers**: A dedicated group of testers will be responsible for testing the app's functionality across different devices
+and platforms.
+
+### Communication and Collaboration
+
+The project will use GitHub for version control, project tracking, and collaboration. All key communication regarding
+project updates, issues, and tasks will be documented and shared through this platform.
+
+## Risk Management
+
+Possible risks include:
+
+**Delayed Development**: Due to team members’ academic commitments, a fixed daily work schedule is in place to ensure
+consistent progress.<br>
+**Data Accuracy**: Ensuring the accuracy of fuel price and location data is critical to the app's functionality.
+
+## Getting Started
+
+To run the project locally, clone the repository:
+
+```bash
+git clone https://github.com/compairifuel/compairifuel.git
+```
+
+For detailed setup and installation instructions, refer to the [INSTALL.md](INSTALL.md) file.
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details.
+
+## Contributors
+
+- [Mino03947](Mino03947)
+- [Jaronline](Jaronline)
+- [Altergith](Altergith)
```

Comment:
When I click on these links I see a 404 page.
Should this link to our github profiles?